### ISSUE TITLE
Zone 0.6.25 Issue with Mongoose Again

### ISSUE THREAD

**Issue by antonyRoberts** [angular/universal-starter#188]
Updated zone.js to version 0.6.25 and the same issue as with moving to 0.6.23 occurs #164 

P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:140
                        throw error;
                        ^

> TypeError: Cannot read property 'filter' of undefined
    at zoneAwareEventListeners (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:772:13)
    at Db.listeners (eval at createNamedFn (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:785:17), <anonymous>:3:36)
    at Server.listener (P:\UniversalSports\node_modules\mongoose\node_modules\mongodb\lib\db.js:1790:15)
    at Server.g (events.js:286:16)
    at ZoneDelegate.invokeTask (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:236:37)
    at Zone.runTask (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:136:47)
    at Server.ZoneTask.invoke (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:304:33)
    at emitTwo (events.js:106:13)
    at Server.emit (events.js:191:7)
    at Server.connectHandler (P:\UniversalSports\node_modules\mongoose\node_modules\mongodb\lib\server.js:329:10)
    at Server.g (events.js:286:16)
    at ZoneDelegate.invokeTask (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:236:37)
    at Zone.runTask (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:136:47)
    at Server.ZoneTask.invoke (P:\UniversalSports\node_modules\zone.js\dist\zone-node.js:304:33)
    at emitOne (events.js:96:13)
    at Server.emit (events.js:188:7)
    at P:\UniversalSports\node_modules\mongoose\node_modules\mongodb-core\lib\topologies\server.js:283:14
    at P:\UniversalSports\node_modules\mongoose\node_modules\mongodb-core\lib\connection\pool.js:436:18
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
[nodemon] app crashed - waiting for file changes before starting...

**Comment by jeffwhelpley** [comment]
@antonyRoberts no need to open a new issue. I am working on prioritizing all issues now and we just need one open issue. Please add comments to #164.
